From b45eb8f62c15d41cf156245548e8004e4b4809d3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Matt=C3=A9o=20Rossillol=E2=80=91=E2=80=91Laruelle?=
 <beatussum@protonmail.com>
Date: Fri, 24 Jul 2020 14:01:48 +0200
Subject: [PATCH] drop `cppunit-config` in favor of `virtual/pkgconfig`
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The upstream project of the `dev-util/cppunit` dependency has decided
to switch to `virtual/pkgconfig`.

See-also: https://cgit.freedesktop.org/libreoffice/cppunit/commit/?id=c81afa737307ef525258cfae24eb3f2845ce7208
Signed-off-by: Mattéo Rossillol‑‑Laruelle <beatussum@protonmail.com>
---
 configure.ac | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index d98e4dd..a2789d9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -113,9 +113,8 @@ AC_ARG_WITH([unit-test],
 if test "x$with_unit_test" = "xyes"
 then
   AC_MSG_NOTICE(enabled cppunit test)
-  AM_PATH_CPPUNIT([1.12.0],
-                  [],
-                  [AC_MSG_ERROR([*** CppUnit not found! ***])])
+  PKG_CHECK_MODULES(CPPUNIT,
+                    [cppunit > 1.12.0])
   AM_CONDITIONAL(CPPUNIT_TEST, [test "1" = "1"])
 else
   AM_CONDITIONAL(CPPUNIT_TEST, [test "1" = "0"])
-- 
2.26.2

